model/transformers/dffml_model_transformers/classification/classification_model.py

```diff
@@ -160,6 +166,15 @@ class HFClassificationModelConfig:
     local_rank: int = field(
         "For distributed training: local_rank", default=-1,
     )
+    dataloader_drop_last: bool = field(
+        "Drop the last incomplete batch if the length of the dataset is not divisible by the batch size",
+        default=False,
+    )
+
+    def to_json_string(self):
+        config_dict = dataclasses.asdict(self)
```

Comment:
Let's use export just in case we ever contain any dffml types

https://intel.github.io/dffml/master/api/util/data.html#dffml.util.data.export

```suggestion
        config_dict = export(self)
```